Packs/Kenna/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1 @@
+Fixed testing for the Kenna pack.
```

Comment:
we changed only the test playbook, not sure we want rn. @amshamah419 how do we do that? `-`?

Reply:
Right now RN will fail when they are empty. Let's use a simple description. I'm hoping to have this issue fixed in the future.